pulseguardian/web.py

```diff
@@ -16,6 +17,7 @@
                    session,
                    g,
                    redirect,
+                   flash,
```

Comment:
Might as well sort this all while you're here. Also I've come to favour this style:

```
from flask import (
    flash,
    Flask,
    ...
)
```